frankDoc/src/test/resources/doc/examplesExpected/sequence.xsd

```diff
@@ -92,6 +97,16 @@
       <xs:pattern value="((\+|-)?[0-9]+)|($\{[^\}]+\})" />
     </xs:restriction>
   </xs:simpleType>
+  <xs:attribute name="active">
+    <xs:annotation>
+      <xs:documentation>If defined and false, then this element and all its children are ignored</xs:documentation>
+    </xs:annotation>
+    <xs:simpleType>
+      <xs:restriction base="xs:string">
+        <xs:pattern value="\!?(($\{[^\}]+\})|((t|T)(r|R)(u|U)(e|E))|((f|F)(a|A)(l|L)(s|S)(e|E)))" />
```

Comment:
Moet het niet [t|T] zijn ipv (t|T) ? `[]` is een characterset en `()` een capturing group.

Reply:
Ik denk dat zowel de versie die ik hier heb als jouw voorstel werken. Ik heb mijn versie getest door een example.xml te maken in VSCode die refereert aan de XSD. Ik heb varianten van true, True, ed geprobeerd en gezien dat het gedrag is wat we willen.

Reply:
Met de [] hoef je geen | meer tussen karakters te hebben, het is al een OR.

Vraag die ik wel heb, is het daadwerkelijk nodig om alle upper en lower case varianten van true en false te ondersteunen? De frankBoolean ondersteund alleen lowercase, en voor active lijkt het mij voldoende om dat gelijk te houden. Daarnaast zal de waarde voor active bijna altijd vanuit een property ingelezen worden, waardoor het mij overbodig lijkt alle cases te ondersteunen. active.xsl zou ik wel case-insensitive houden zoals die nu is, maar in de xsd zou ik het simpel houden en alleen lowercase accepteren.

Reply:
Ik zou de voorkeur geven aan gebruik van (?i), waarmee je de regex case insensitive maakt.
Dan wordt het zoiets als:
\!?(?i)(($\{[^\}]+\})|(true)|(false)))

Reply:
Gerrit, wat je vraagt kan waarschijnlijk in Java maar niet in XML Schema. Zie https://stackoverflow.com/questions/30428591/case-insensitive-regex-in-xsd. Ben je het met Ricardo eens dat we niet alle upper/lower case varianten van true en false willen ondersteunen, maar alleen true, TRUE, false, FALSE? Ik zou het zelf liever hetzelfde houden als in active.xsl, dus wel alle combinaties uppercase/lowercase ondersteunen. Als je wilt, dan kan ik de capturing groups vervangen door [] expressies zodat de |-symbolen weg kunnen. Wat zal ik doen?

Reply:
Gerrit, ik heb je door de telefoon gevraagd wat we gaan doen. Ik ga alleen de regex versimpelen door capture groups door character groups te vervangen.

Reply:
Ik heb de capture groups net door character groups vervangen